EIPS/eip-4519.md

```diff
@@ -209,12 +210,13 @@ This EIP proposes using the Smart Contract to verify the mutual authentication p
 This EIP proposes the attributes timestamp (to register in the blockchain whenever the physical asset checks the tie with its token) and timeout (to register the maximum delay time established for the physical asset to prove again the tie). These attributes avoid a malicious owner or user could use the asset endlessly.
```

Comment:
```suggestion
This EIP proposes the attributes timestamp (to register in the blockchain whenever the physical asset checks the tie with its token) and timeout (to register the maximum delay time established for the physical asset to prove again the tie). These attributes avoid a malicious owner or user could use the asset endlessly.
When the asset calls `updateTimestamp`, the smart contract must call `block.timestamp`, that provide current block timestamp as seconds since unix epoch. For this reason, `timeout`  must be provided in seconds.
```